### ISSUE TITLE
Blender crash on Action that scales and moves UV islands

### ISSUE THREAD

**Issue by baroquedub** [InamuraJIN/ActionRecorder#49]
Blender crashes when trying to run an action that attempts to scale and  move selected UV islands.

The action was recorded using keyboard shortcuts (e.g. S 0.75 [enter] G X -0.12 [enter] G Y 0.225 [enter])

**To Reproduce**
Enter UV edit mode
Select UVs
Create Action
Go into record
Type S 0.5 [enter] G X 0.25 [enter] G Y -0.5 [enter]
Stop record
Reset UVs to previous (just for debugging purposes)
Press 'Play' on Action

**Expected behavior**
In the UV editor, with some UV islands selected, they should be scaled and moved in the same way as the recorded Action
Instead, Blender crashes

**Screenshots**
Video of process: https://youtu.be/mBA70D6ps7c (unlisted)

**Version Information:**
 - Add-on Version 4.07
 - Blender Version 3.4.1
 - OS: Windows10

**Additional context**
Crash Log provided:
[box_uv.crash.txt](https://github.com/InamuraJIN/ActionRecorder/files/11102987/box_uv.crash.txt)

**Comment by RivinHD** [comment]
Here is a temporary solution:
you can add your commands to a script and run this script as an action

**Instructions:**
```py
import bpy
# Your recording / macros
bpy.ops.transform.resize(value=(0.5, 0.5, 0.5), orient_type='GLOBAL', orient_matrix=((1, 0, 0), (0, 1, 0), (0, 0, 1)), orient_matrix_type='GLOBAL', mirror=False, use_proportional_edit=False, proportional_edit_falloff='SMOOTH', proportional_size=1, use_proportional_connected=False, use_proportional_projected=False, snap=False, snap_elements={'INCREMENT'}, use_snap_project=False, snap_target='CLOSEST', use_snap_self=True, use_snap_edit=True, use_snap_nonedit=True, use_snap_selectable=False)
bpy.ops.transform.translate(value=(0.25, -0.25, 0), orient_axis_ortho='X', orient_type='GLOBAL', orient_matrix=((1, 0, 0), (0, 1, 0), (0, 0, 1)), orient_matrix_type='GLOBAL', mirror=False, use_proportional_edit=False, proportional_edit_falloff='SMOOTH', proportional_size=1, use_proportional_connected=False, use_proportional_projected=False, snap=False, snap_elements={'INCREMENT'}, use_snap_project=False, snap_target='CLOSEST', use_snap_self=True, use_snap_edit=True, use_snap_nonedit=True, use_snap_selectable=False)
```
Copy this and paste it into a Text in the TextEditor (the name of the Text does not matter)
![image](https://user-images.githubusercontent.com/58261670/228986514-a7db1210-d98d-473b-a258-948ecf402ded.png)
now you can add this text as a script event in ActionRecorder
![image](https://user-images.githubusercontent.com/58261670/228986715-d59da429-b895-4113-89df-b033e66e90ac.png)

**Comment by baroquedub** [comment]
Can confirm that temporary solution of using script events works well. Thank you!

**Comment by RivinHD** [comment]
ok, this fixed now in Version [4.0.8](https://github.com/InamuraJIN/ActionRecorder/releases/tag/v4.0.8)

further information on https://projects.blender.org/blender/blender/issues/106105
